src/configure/configure.ts

```diff
@@ -478,12 +478,12 @@ class Orchestrator {
                 throw error;
             }
         }
-        else if (!!node.cloudResource && node.cloudResource.nodeType === 'cluster') {
-            this.inputs.subscriptionId = node.cloudResource.subscription.subscriptionId;
+        else if (node.resource.nodeType === "cluster") {
```

Comment:
This function should bifurcate based on whether it is a resource right click or folder right click and that should happen based on the type of `node` first.

Reply:
The bifurcation is at ``` analyzeNode ``` function. 
It eneters this section only if it's a right click on scenario.